app/web/src/features/auth/AuthPage.tsx

```diff
@@ -26,66 +19,87 @@ import { COUCHERS } from "../../constants";
 
 const useStyles = makeStyles((theme) => ({
   aboutUs: {
-    color: theme.palette.common.white,
-    marginTop: theme.spacing(2),
-  },
-  aboutUsLink: {
-    color: theme.palette.common.white,
     display: "block",
+    color: theme.palette.common.white,
+    textAlign: "center",
+    paddingTop: theme.spacing(1),
+    fontWeight: 500,
+    fontSize: "1rem",
+    marginTop: theme.spacing(1),
+    lineHeight: 1,
     [theme.breakpoints.up("md")]: {
-      color: theme.palette.common.white,
-      fontWeight: 500,
       fontSize: "1.25rem",
-      marginRight: theme.spacing(3),
-      padding: theme.spacing(1),
     },
   },
   authPage: {
-    alignItems: "flex-end",
-    backgroundColor: theme.palette.common.white,
-    backgroundImage: `url(${MobileAuthBg})`,
+    display: "flex",
+    flexDirection: "column",
+
+    background: `linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), url("${MobileAuthBg}")`,
     backgroundPosition: "top center",
     backgroundRepeat: "no-repeat",
     backgroundSize: "cover",
-    boxSizing: "border-box",
-    display: "flex",
     height: "100vh",
-    justifyContent: "center",
+    width: "100%",
     padding: `${theme.spacing(1, 4)}`,
+    paddingBottom: 0,
     [theme.breakpoints.up("md")]: {
-      alignItems: "flex-start",
-      backgroundImage: `url(${DesktopAuthBg})`,
+      background: `url(${DesktopAuthBg})`,
       backgroundSize: "cover",
-      flexDirection: "column",
-      padding: 0,
-      width: "100%",
     },
+    [theme.breakpoints.down("sm")]: {},
   },
   button: {
     width: "45%",
   },
   content: {
-    display: "flex",
+    flexGrow: 1,
+    color: theme.palette.common.white,
     flexDirection: "column",
-    height: "50vh",
-    justifyContent: "center",
+    display: "flex",
+    justifyContent: "flex-end",
     textAlign: "center",
     [theme.breakpoints.up("md")]: {
-      height: "100%",
-      margin: "0 auto",
-      width: theme.breakpoints.values.md,
+      justifyContent: "center",
+      textAlign: "left",
+      width: "72%",
     },
   },
-  desktopNavigation: {
-    display: "flex",
+  title: {
+    ...theme.typography.h1,
+    [theme.breakpoints.up("md")]: {
+      fontSize: "4rem",
+      lineHeight: "1.15",
+    },
   },
-  mobileIntroduction: {
-    color: theme.palette.common.white,
+  subtitle: {
+    marginTop: theme.spacing(1),
+    [theme.breakpoints.up("md")]: {
+      display: "inline-block",
+      marginTop: theme.spacing(4),
+      fontSize: "1.5rem",
+      lineHeight: "1.15",
+    },
   },
-  introduction: {
-    color: theme.palette.common.white,
-    textAlign: "left",
-    width: "72%",
+  underline: {
+    borderTop: `5px solid ${theme.palette.primary.main}`,
+    boxShadow: "0px 4px 4px rgba(0, 0, 0, 0.25)",
```

Comment:
I wonder if it's worth adding stuff like this into the `shadow` part of our theme. Or maybe not since this styling is so distinct from the rest of our authenticated app?

Reply:
Yeah it's not a material shadow and not used anywhere else so let's leave it